issue_template.md

```diff
@@ -1,29 +1,41 @@
-### Summary
-
--   What does this package do? (explain in 50 words or less):
+Submitting Author: Name (@github_handle)  
+Repository:   
```

Comment:
maybe specify url, `Repository URL: `

Reply:
Sure. But even if it were just repository, what else could it be?